ext/pdo/tests/gh16703.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+GH-16703: Memory leak of setFetchMode()
+--EXTENSIONS--
+pdo
+--SKIPIF--
+<?php
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+require_once $dir . 'pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+if (getenv('REDIR_TEST_DIR') === false) putenv('REDIR_TEST_DIR='.__DIR__ . '/../../pdo/tests/');
+class TestStmt extends PDOStatement
```

Comment:
```suggestion
#[AllowDynamicProperties]
class TestStmt extends PDOStatement
```

Reply:
Thanks, I tried forcing the column names to be lowercase :)

Reply:
Yeah, of course!